antarest/study/storage/abstract_storage_service.py

```diff
@@ -80,6 +80,8 @@ def get_study_information(
             id=study.id,
             name=study.name,
             version=study.version,
+            author=study.author,
```

Comment:
We should take the field author from additional_data.
This one from study seems never used, we should remove it, but not in that PR.